### ISSUE TITLE
Two-finger scroll when TalkBack is enabled doesn't work

### ISSUE THREAD

**Issue by Hixie** [flutter/flutter#1721]
e.g. try to scroll the stocks app or the demo list in the galley app when TalkBack is enabled (using the two-finger drag gesture).

**Comment by Hixie** [comment]
I'm guessing we're getting a weird set of events from the system. It looks like they send us both fingers' events while you're two-finger-touching, but then eat the events when you lift one?

**Comment by goderbauer** [comment]
It seems to be working just fine in the Stocks app on my Nexus 6P. Closing this as it is old and probably got fixed along the way somewhere.
